cli/medperf/commands/compatibility_test/validate_params.py

```diff
@@ -0,0 +1,106 @@
+from medperf.exceptions import InvalidArgumentError
+
+
+class CompatibilityTestParamsValidator:
+    """Validates the input parameters to the CompatibilityTestExecution class"""
+
+    def __validate_cubes(self):
+        if not self.model and not self.benchmark_uid:
```

Comment:
This is another example of what I meant by highly coupled implementation. This class by itself doesn't have any definition of the properties being tested, so reading this class by itself makes it appear like it will always raise exceptions